packages/client/hmi-client/src/types/workflow.ts

```diff
@@ -1,3 +1,11 @@
+export enum WorkflowStatus {
```

Comment:
i think we need to add a status for a node that has not been triggered to run?

Reply:
I was thinking along the same lines, but I don't want to have a plethora of different status code where some of them are kind of similar.....I think for now a node that has not been run can use `invalid` and we can revisit adding more enum-tokens as needed.